go/cmd/paddlecloud/paddlecloud.go

```diff
@@ -18,10 +18,12 @@ func main() {
 	subcommands.Register(&paddlecloud.GetCommand{}, "")
 	subcommands.Register(&paddlecloud.KillCommand{}, "")
 	subcommands.Register(&paddlecloud.SimpleFileCmd{}, "")
-	subcommands.Register(&pfsmod.LsCmd{}, "")
-	subcommands.Register(&pfsmod.CpCmd{}, "")
-	subcommands.Register(&pfsmod.RmCmd{}, "")
-	subcommands.Register(&pfsmod.MkdirCmd{}, "")
+	subcommands.Register(&paddlecloud.RegistryCmd{}, "")
+	subcommands.Register(&paddlecloud.DeleteCommand{}, "")
+	subcommands.Register(&pfsmod.LsCmd{}, "PFS")
```

Comment:
The second argument of `subcommands.Register` is [group](https://github.com/google/subcommands/blob/master/subcommands.go#L95), if we add a subcommand with a specify a group name, these subcommands will be explained with a group name before, such as:
```bash
Subcommands:
	commands         list all command names
	delete           Delete the specify resource.
	file             Simple file operations.
...

Subcommands for PFS:
	cp               uoload or download files
	ls               List files on PaddlePaddle Cloud
...
```